### PR TITLE
Confirmation Dialogs SideBar🚸

### PR DESCRIPTION
**Pull request by gauri-sharma-cerelabs**

### CODE REVIEW

File: src/ConfirmationDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+} from "@mui/material";
+
+interface ConfirmationDialogProps {
```
Comment (parth-mangtani-cerelabs) [inline src/ConfirmationDialog.tsx]: move this to a separate file

File: src/ConfirmationDialog.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  onClose,
+  onConfirm,
+  title,
+  content,
+}) => {
+  return (
+    <Dialog
+      open={open}
+      onClose={onClose}
+      aria-labelledby="confirmation-dialog-title"
```
Comment (parth-mangtani-cerelabs) [inline src/ConfirmationDialog.tsx:27]: these labels will need to be generic

File: src/ConfirmationDialog.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      aria-describedby="confirmation-dialog-description"
+    >
+      <DialogTitle id="confirmation-dialog-title">{title}</DialogTitle>
```
Comment (parth-mangtani-cerelabs) [inline src/ConfirmationDialog.tsx]: we dont need an id here

File: src/ConfirmationDialog.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+      <DialogContent>
+        <DialogContentText id="confirmation-dialog-description">
```
Comment (parth-mangtani-cerelabs) [inline src/ConfirmationDialog.tsx]: we dont need an id here

File: src/ConfirmationDialog.tsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+          {content}
+        </DialogContentText>
+      </DialogContent>
+      <DialogActions>
+        <Button onClick={onClose} color="primary">
```
Comment (parth-mangtani-cerelabs) [inline src/ConfirmationDialog.tsx]: please change the color for this to be default

File: src/PromptEngineeringSidebar.tsx [hunk 6]
```diff
@@ -166,71 +165,104 @@ function PromptEngineeringSidebar(props: {
...
-              <AccordionSummary
-                expandIcon={<ExpandMoreIcon />}
-                aria-controls="panel1-content"
-                id="panel1-header"
+
+        {props.examples.map((example, idx) => (
+          <Accordion key={idx}>
+            <AccordionSummary
+              expandIcon={<ExpandMoreIcon />}
+              aria-controls="panel1-content"
```
Comment (parth-mangtani-cerelabs) [inline src/PromptEngineeringSidebar.tsx]: we should include the key in the id and aria-controls

**Review by parth-mangtani-cerelabs — CHANGES_REQUESTED** [review]

File: src/ConfirmationDialog.tsx (hunk 2 above)
Comment (gauri-sharma-cerelabs) [reply src/ConfirmationDialog.tsx:27]: did thaton line 20 and 21

File: src/ConfirmationDialog.tsx (hunk 1 above)
Comment (gauri-sharma-cerelabs) [reply src/ConfirmationDialog.tsx]: moved it to PromptEngineeringSidebar.ts

File: src/ConfirmationDialog.tsx (hunk 3 above)
Comment (gauri-sharma-cerelabs) [reply src/ConfirmationDialog.tsx]: removed

File: src/ConfirmationDialog.tsx (hunk 4 above)
Comment (gauri-sharma-cerelabs) [reply src/ConfirmationDialog.tsx]: removed

File: src/ConfirmationDialog.tsx (hunk 5 above)
Comment (gauri-sharma-cerelabs) [reply src/ConfirmationDialog.tsx]: yes made said changes

File: src/PromptEngineeringSidebar.tsx (hunk 6 above)
Comment (gauri-sharma-cerelabs) [reply src/PromptEngineeringSidebar.tsx]: included

**Review by parth-mangtani-cerelabs — APPROVED** [review]

**Pull request merged**